### PR TITLE
Small CR iperf3 CR change & docs update

### PR DESCRIPTION
**Pull request by sjug**
### Description
Some of the links in the docs were fixed previously by https://github.com/cloud-bulldozer/benchmark-operator/pull/616, however, there was a few misses.
Some of the docs were inconsistent in their sample sections.
Discovered while taking a look at https://github.com/cloud-bulldozer/benchmark-operator/issues/642.

### Fixes
- Small change to comment out length of buffer in iperf3 test, as 128K is the default for TCP but hardcoding the value breaks UDP.
- Minor corresponding changes to docs.

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM Thanks for reviewing and fixing the issues!

**Comment by rsevilla87** [comment]
Hey @sjug , this PR will break the release workflow https://github.com/cloud-bulldozer/benchmark-operator/blob/master/.github/workflows/release.yml. 
The current implementation sets VERSION to the current tag or branch name, this workflow builds container images when new code is commited in the master branch or a new tag is pushed to the repo.

In case you're working on a new branch and you want to deploy benchmark-operator you can set IMG and point it to the desired image, otherwise it will try to deploy quay.io/cloub-bulldozer/benchmark-operator:<branch-name>.

```
IMG=quay.io/cloub-bulldozer/benchmark-operator:latest make deploy
```

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
```
Comment (rsevilla87) [inline config/manager/kustomization.yaml:8]: I think you commited this file by mistake, as this file is modified by kustomize when you deploy the operator

Comment (sjug) [reply config/manager/kustomization.yaml:8]: No mistake, I'm aware that it's modified by kustomize when the operator is deployed. However as the code stands this is what would be in the file.

If changes are made this file will still be modified. However if one were to just checkout the code and `make deploy` this file would not be modified. I feel that's a reasonable expectation if one doesn't modify the code otherwise.

**Comment by sjug** [comment]
> Hey @sjug , this PR will break the release workflow https://github.com/cloud-bulldozer/benchmark-operator/blob/master/.github/workflows/release.yml.
> The current implementation sets VERSION to the current tag or branch name, this workflow builds container images when new code is commited in the master branch or a new tag is pushed to the repo.
> 
> In case you're working on a new branch and you want to deploy benchmark-operator you can set IMG and point it to the desired image, otherwise it will try to deploy quay.io/cloub-bulldozer/benchmark-operator:.
> 
> ```
> IMG=quay.io/cloub-bulldozer/benchmark-operator:latest make deploy
> ```

Interesting, thanks for the heads up. I figured it was used for something, just not sure what.
I guess the issue is essentially that this `VERSION` is tailored only for CI.

The `master` branch in this original repo does not have tags so therefore it for some reason becomes `latest`.
However on any forks, the master branch does have a tag, and therefore you get something like: 
```
quay.io/cloud-bulldozer/benchmark-operator:v0.1-31-g2d3dc8d
```

I feel like it makes more sense for CI to set the version explicitly rather than requiring the user to do so. What do you think?

File: Makefile [hunk 2]
```diff
@@ -40,9 +40,9 @@ ARCH ?= $(shell uname -m | sed 's/x86_64/amd64/')
 # Image URL to use all building/pushing image targets
 REGISTRY ?= quay.io
 ORG ?= cloud-bulldozer
-# Get the current branch/tag name
+# Get the current branch name
 # In case this is the master branch, rename it to latest
-VERSION ?= $(shell git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g')
+VERSION ?= $(shell git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g')
```
Comment (rsevilla87) [inline Makefile:45]: I don't understand this change. The original code was able to set VERSION to something like v1.1 if you are in the the tag v.1.1 or failback to set VERSION to the current branch name (in case of master this is updated to latest).

You have drop detecting the tag name in your implementation, e.g.:

Output using the original implementation:
```shell
rsevilla@wonderland ~/labs/ripsaw ((v0.1)) $ git --version
git version 2.31.1
rsevilla@wonderland ~/labs/ripsaw ((v0.1)) $ git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g'
v0.1
rsevilla@wonderland ~/labs/ripsaw (master) $ git checkout e2e-testing
Switched to branch 'e2e-testing'
rsevilla@wonderland ~/labs/ripsaw (e2e-testing) $ git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g'
e2e-testing
```

Comment (sjug) [reply Makefile:45]: Yes indeed, I too can understand the bash involved. Let me rephrase my previous message; the issue was/is that the "original code" would resolve to a tag even on an untagged branch such as `master`.
```
$ git branch
* master
$ git describe --tags
v0.1-31-g2d3dc8d
```

This isn't the case on other forks, so it must be an issue on just mine. I'll revert the change.

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (rsevilla87) [reply config/manager/kustomization.yaml:8]: Ok, seems to work, I thought this file wouldn't be modified once kustomize had changed it. :)

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Comment by sjug** [comment]
The way we set `VERSION` seems to be [breaking the prow CI jobs](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-kubernetes-release-4.8-configmap-scale).
